src/frontend/vite.config.ts

```diff
@@ -51,8 +52,36 @@ export default defineConfig({
   base: '',
   build: {
     manifest: true,
-    outDir: '../../src/backend/InvenTree/web/static/web',
-    sourcemap: is_coverage
+    // outDir: '../../src/backend/InvenTree/web/static/web',
+    // TODO: outDir should be set to the correct path,
+    // TODO: depending on whether we are building library or UI?
```

Comment:
Maybe we can split up the vite config files into multiple or use some arguments supplied by the defineConfig function if we pass a function there to determine the target. ref: https://vite.dev/config/

Reply:
Ok, that sounds pretty reasonable :)